management-app/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.4/apache-maven-3.5.4-bin.zip
```

Comment:
Once you add more Maven submodules to your project you can explore the option to have this wrapper at a project level instead of having it in the submodules.

Reply:
Thanks, I will include this in the PR where I integrate both maven submodules so I can test it actually working.